### PR TITLE
Checked a few games on steam

### PR DESCRIPTION
**Pull request by daddeltrotter**

### CODE REVIEW

File: gamedb.yaml [hunk 1]
```diff
@@ -5287,3 +5287,39 @@
...
+  compat_tool: GE-Proton8-15
+  status: verified
+
+- name: 'Master Levels for Doom II'
+  id: 9160
+  platform: steam
+  compat_tool: boxtron
+
+# Controller doesn't work in native version
+- name: 'Savent - Ascent'
```
Comment (alkazar) [inline gamedb.yaml:5312]: The game name is "Savant - Ascent"

File: gamedb.yaml [hunk 2]
```diff
@@ -5287,3 +5287,39 @@
...
+  id: 410110
+  compat_tool: proton-stable
+  status: playable
+  notes:
+   - Controller needs to be enabled in the game
+
+- name: 'The Metronomicon: Slay The Dance Floor'
+  id: 416790
+  platform: steam
+  compat_tool: GE-Proton8-15
```
Comment (alkazar) [inline gamedb.yaml:5303]: Is this specific version of GE Proton required? 8-25 is the latest that is available.

File: gamedb.yaml (hunk 1 above)
Comment (daddeltrotter) [reply gamedb.yaml]: Yes, Corrected the typo.

File: gamedb.yaml (hunk 2 above)
Comment (daddeltrotter) [reply gamedb.yaml]: Changed it to 8-25
When I tested it, only 8-21 was available which did not work.

**Comment by alkazar** [comment]
Thank you!

**Pull request merged**